style.css

```diff
@@ -86,3 +95,160 @@ main {
 .arrow-icon {
   text-align: center;
 }
+
+.flex {
+  display: flex;
+  padding-bottom: 20px;
+  width: 100%;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.works-section-title {
+  text-align: center;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.works-grid-container {
+  display: grid;
+  gap: 24px;
+  justify-content: center;
+  width: 86%;
+  margin: 62px auto;
+}
+
+.card img {
+  width: 327px;
+  max-width: 100%;
+  height: 252px;
+}
+
+.card {
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+button {
+  cursor: pointer;
+}
+
+.card-details {
+  align-items: center;
+  gap: 12px;
+  color: #172b4d;
+}
```

Comment:
- Some issues I am mentioning below regarding this section. Kindly fix them.
- [ ] There shouldn't be spaces/padding on the right side of the images. I have marked red on the below picture. 
- [ ] I can see some extra dots on the left side of the list of items like `. Ruby on Rails, `. CSS`, etc. Kindly remove them.  You use list-style to fix this and consider reading the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) for how to do this.
- [ ] Use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design:
- ![image](https://user-images.githubusercontent.com/60083437/227739727-e7e9b130-dcfd-430b-a1ba-b7db9b0bcb0c.png)